unity-renderer/Assets/DCLPlugins/SignupHUDPlugin/SignupHUDPlugin.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Cysharp.Threading.Tasks;
+using DCL;
+using DCL.Providers;
+using SignupHUD;
+
+namespace DCLPlugins.SignupHUDPlugin
+{
+    public class SignupHUDPlugin : IPlugin
+    {
+        private const string SIGNUP_HUD = "SignupHUD";
+
+        private SignupHUDController controller;
+
+        public SignupHUDPlugin()
+        {
+            Initialize().Forget();
+        }
+
+        private async UniTaskVoid Initialize()
```

Comment:
I have a feeling somewhere here should be a `CancellationToken`, similar to `InitializeAsync` from `IService`?

Reply:
+1, but perhaps it out of the scope of this pr